<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1462</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame{
background:rgb(242, 226, 177);

}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>890</width>
         <height>80</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>890</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>890</width>
         <height>80</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnclose">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background: rgb(241, 0, 0);
    border-radius: 30px;
    font: 13px &quot;Yu Gothic UI Semibold&quot;;
    border: 2px solid rgb(189, 179, 149);  
}

QPushButton:hover {
    background: rgb(230, 220, 220); 
    border: 2px solid rgb(50, 150, 255);  
}</string>
          </property>
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>800</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>800</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
font-size: 30px;
font: &quot;System&quot;;
}</string>
          </property>
          <property name="text">
           <string>Algoritmo de analisis para prediccion de abandono escolar</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>470</y>
         <width>800</width>
         <height>80</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>80</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="btnXG">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background: rgb(246, 240, 240);
    border-radius: 15px;
    font: 13px &quot;Yu Gothic UI Semibold&quot;;
    border: 2px solid rgb(189, 179, 149);  
}

QPushButton:hover {
    background: rgb(230, 220, 220); 
    border: 2px solid rgb(50, 150, 255);  
}</string>
          </property>
          <property name="text">
           <string>Activar Modelo XGBOOST</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnLoadData">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background: rgb(246, 240, 240);
    border-radius: 15px;
    font: 13px &quot;Yu Gothic UI Semibold&quot;;
    border: 2px solid rgb(189, 179, 149);  
}

QPushButton:hover {
    background: rgb(230, 220, 220); 
    border: 2px solid rgb(50, 150, 255);  
}</string>
          </property>
          <property name="text">
           <string>Cargar datos</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btnRF">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background: rgb(246, 240, 240);
    border-radius: 15px;
    font: 13px &quot;Yu Gothic UI Semibold&quot;;
    border: 2px solid rgb(189, 179, 149);  
}

QPushButton:hover {
    background: rgb(230, 220, 220); 
    border: 2px solid rgb(50, 150, 255);  
}</string>
          </property>
          <property name="text">
           <string>Activar Modelo Random Forest</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTableWidget" name="table">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>860</width>
         <height>350</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>860</width>
         <height>350</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>860</width>
         <height>350</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableView{
background: white;
    border: 2px solid rgb(189, 179, 149);  
}</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_4">
       <property name="geometry">
        <rect>
         <x>910</x>
         <y>90</y>
         <width>500</width>
         <height>370</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>370</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>370</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
background:rgb(242, 226, 177);

}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextBrowser" name="textArea">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>350</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextBrowser{

font: 15px;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>920</x>
         <y>10</y>
         <width>441</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font-size: 30px;
font: &quot;System&quot;;
}</string>
       </property>
       <property name="text">
        <string>Resultados del Entrenamiento: </string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_5">
       <property name="geometry">
        <rect>
         <x>910</x>
         <y>470</y>
         <width>500</width>
         <height>80</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>80</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btncleardata">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background: rgb(246, 240, 240);
    border-radius: 15px;
    font: 13px &quot;Yu Gothic UI Semibold&quot;;
    border: 2px solid rgb(189, 179, 149);  
}

QPushButton:hover {
    background: rgb(230, 220, 220); 
    border: 2px solid rgb(50, 150, 255);  
}</string>
          </property>
          <property name="text">
           <string>Limpiar datos</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
